include/wren/frame_buffer.h

```diff
@@ -29,7 +29,7 @@ void wr_frame_buffer_setup(WrFrameBuffer *frame_buffer);
 void wr_frame_buffer_blit_to_screen(WrFrameBuffer *frame_buffer);
 
 /* The data written to 'data' will be in the format of output texture number 'index' */
-void wr_frame_buffer_copy_contents(WrFrameBuffer *frame_buffer, int index, void *data);
+void wr_frame_buffer_copy_contents(WrFrameBuffer *frame_buffer, int index, void *data, int offset = 0, int size = 0);
```

Comment:
I am surprised this works. Normally, in pure C, we cannot define default parameters values for functions...
```suggestion
void wr_frame_buffer_copy_contents(WrFrameBuffer *frame_buffer, int index, void *data, int offset, int size);
```